server/protocols/jmap/src/main/java/org/apache/james/jmap/crypto/JamesSignatureHandler.java

```diff
@@ -64,10 +65,15 @@
     @Override
     public void init() throws Exception {
         KeyStore keystore = KeyStore.getInstance(JKS);
-        InputStream fis = fileSystem.getResource(jmapConfiguration.getKeystore());
-        keystore.load(fis, jmapConfiguration.getSecret().toCharArray());
+        String keystorRessource = jmapConfiguration.getKeystore();
+        InputStream fis = fileSystem.getResource(keystorRessource);
+        if (fis == null) {
+            throw new FileNotFoundException(keystorRessource);
+        }
+        char[] charArray = jmapConfiguration.getSecret().toCharArray();
```

Comment:
charArray is the type, not the name (: 
what about `char[] secret` ?
BTW, getSecret should already return a `char[]` to protect about accidental toString